services/repository/src/main/java/org/sagebionetworks/repo/web/controller/BaseControllerExceptionHandlerAdvice.java

```diff
@@ -933,7 +933,7 @@ ErrorResponse handleInvalidTableQueryFacetColumnRequestException(InvalidTableQue
 	ErrorResponse handleNoHandlerFoundException(NoHandlerFoundException ex, HttpServletRequest request){
 		return handleException(ex,
 				request,
-				ex.getHttpMethod() + " " + ex.getRequestURL() + " was not found. Please reference API documentation at https://docs.synapse.org/rest/",
+				ex.getHttpMethod() +" was not found. Please reference API documentation at https://docs.synapse.org/rest/",
```

Comment:
We should leave the getRequestURL() since getHttpMethod() would be very confusing. We should also comment that there is a bug where getHttpMethod() returns the path instead of the verb. The error message can be built using the request parameter instead.